.github/workflows/e2e_ci.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 
       - name: Install deps
         run: |
-          sudo apt install -y --no-install-recommends clang gcc llvm build-essential cmake python3 quota
+          sudo apt install -y --no-install-recommends clang gcc llvm build-essential cmake python3 python2 quota
```

Comment:
Please port any script to python3 instead, we need python3 now: https://github.com/falcosecurity/libs/pull/2073

Reply:
Already fixed! :)

Reply:
Thank you man!